### PR TITLE
Allow cross site request and add configuration for CORS

### PR DESCRIPTION
**Pull request by chiro-hiro**
## Type

- [x] Feature
- [ ] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/serverless/src/helper/config.ts [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-  LoggerSet,
-  LoggerLoader,
-} from '@orochi-network/framework';
+import { ConfigLoader, TNodeEnv } from '@orochi-network/framework';
 import { createHash } from 'crypto';
 import Joi from 'joi';
 
-LoggerSet(new LoggerLoader('framework', 'debug', 'json'));
-
 interface TApplicationConfig {
```
Comment (SangTran-127) [inline packages/serverless/src/helper/config.ts:6]: should be type

File: packages/serverless/src/helper/config.ts [hunk 2]
```diff
@@ -30,6 +25,14 @@ const configLoader = new ConfigLoader<TApplicationConfig>(
       .update('jwt')
       .update(raw.SERVICE_SECRET)
       .digest('base64');
+    const { hostname, port } = new URL(raw.SERVICE_BIND);
```
Comment (SangTran-127) [inline packages/serverless/src/helper/config.ts:28]: using util url parser

File: packages/serverless/src/apollo/app/user.ts [hunk 3]
```diff
@@ -267,14 +254,18 @@ const userSignOut = authorizeWrapper(
 const userSignUp = publicWrapper(
   SignUpWrapper,
   async (_root: unknown, args: TSignUpWrapper) => {
+    const {
+      signUp: { userData, userName, email },
+      proof,
+    } = args;
```
Comment (SangTran-127) [inline packages/serverless/src/apollo/app/user.ts:260]:
```suggestion
    const {
      signUp: { userData, userName, email },
      proof: { publicKey },
    } = args;
```

File: packages/serverless/src/apollo/app/user.ts [hunk 4]
```diff
@@ -267,14 +254,18 @@ const userSignOut = authorizeWrapper(
...
     return signUpUser(
       {
-        userName: args.signUp.userName,
-        email: args.signUp.email,
-        publicKey: args.proof.publicKey,
+        userName,
+        email,
+        publicKey: proof.publicKey,
```
Comment (SangTran-127) [inline packages/serverless/src/apollo/app/user.ts:265]:
```suggestion
        publicKey,
```

File: packages/serverless/src/helper/config.ts (hunk 2 above)
Comment (chiro-hiro) [reply packages/serverless/src/helper/config.ts:28]: Utilities wasn't installed

File: packages/serverless/src/helper/config.ts (hunk 1 above)
Comment (chiro-hiro) [reply packages/serverless/src/helper/config.ts]: Updated

**Review by SangTran-127 — APPROVED** [review]

**Review by SangTran-127 — APPROVED** [review]

**Pull request merged**